ui-admin/src/forms/FormContentEditor.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { render } from '@testing-library/react'
+import React from 'react'
+
+import { FormContentEditor } from './FormContentEditor'
+
+//This is valid JSON, but invalid survey JSON
```

Comment:
Right now the JSON editor will prevent you from saving or leaving the editor tab if the JSON is invalid. I think the next step to improve this would be to give that same treatment to valid JSON that isn't valid _survey_ JSON.

Reply:
Filed as https://broadworkbench.atlassian.net/browse/JN-539

Didn't want to put it into this PR because I think it's a bit more effort to make sure the edge cases are covered, and the priority with this PR is to prevent irrecoverable states.